daemon/cmd/daemon_main.go

```diff
@@ -804,6 +804,18 @@ func init() {
 	flags.String(option.HubbleListenAddress, "", `An additional address for Hubble server to listen to, e.g. ":4244"`)
 	option.BindEnv(option.HubbleListenAddress)
 
+	flags.Bool(option.HubbleTLSDisabled, false, "Allow Hubble server to run on the given listen address with TLS disabled.")
```

Comment:
why is the option a negative one rather than a positive one?

Reply:
The idea was to imply, just by reading the options, that TLS is always enabled unless explicitly disabled. The idea is also to ensure to fail safe rather than fail open (although the default value passed to `flags.Bool()` kinda mitigate this already).

Reply:
To me it make sense to have a negative flag. Effectively, it forces to disable TLS explicitly as @Rolinh pointed out ("secure by default").

With that said, I think that the in-code logic should use `true` for TLS enabled and `false` for TLS disabled, as `bool` with a WithoutSometing logic are arguably more error-prone to use.